watchfaces-img/funky01.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
After having tried the funky watchface inside an emulated Asteroid, it looks like the svg render pixelated
![screenshot from 2018-04-28 23-50-19](https://user-images.githubusercontent.com/908424/39401173-2f05faca-4b3f-11e8-8e90-efa910b3aa78.png)
![screenshot from 2018-04-28 23-49-59](https://user-images.githubusercontent.com/908424/39401174-2f1d7ac4-4b3f-11e8-9790-f5fb207999dd.png)

Reply:
fixed according to this solution: https://forum.qt.io/topic/11859/solved-rendering-svg-images/
added width to source svg so it renders according to parent size.
        sourceSize.width: parent.width
        sourceSize.height: parent.height

PR update incoming this evening